src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.IO;
+using UnityEditor.iOS.Xcode;
+using UnityEditor.iOS.Xcode.Extensions;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor.iOS
+{
+    internal class SentryXcodeProject : IDisposable
+    {
+        private const string FrameworkName = "Sentry.framework";
+
+        private const string MainPath = "MainApp/main.mm";
+        private const string OptionsPath = "MainApp/SentryOptions.m";
+        private string UnityPackageFrameworkRoot = Path.Combine("Frameworks", "io.sentry.unity");
```

Comment:
It's a bit inconsistent, either go with `/` or use `Path.Combine` on all, like:

```suggestion
        private const string MainPath = Path.Combine("MainApp", "main.mm");
        private const string OptionsPath = Path.Combine("MainApp"," SentryOptions.m");
        private string UnityPackageFrameworkRoot = Path.Combine("Frameworks", "io.sentry.unity");
```